browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchAdapter.java

```diff
@@ -0,0 +1,101 @@
+package com.braintreepayments.browserswitch;
+
+import android.content.Context;
+import android.content.Intent;
+import android.net.Uri;
+
+import androidx.annotation.VisibleForTesting;
+import androidx.fragment.app.Fragment;
+import androidx.fragment.app.FragmentActivity;
+
+public class BrowserSwitchAdapter implements IBrowserSwitchClient {
+
+    static BrowserSwitchAdapter newInstance() {
+        return newInstance(
+                BrowserSwitchClient.newInstance(), BrowserSwitchConfig.newInstance());
+    }
+
+    @VisibleForTesting
+    static BrowserSwitchAdapter newInstance(BrowserSwitchClient simpleClient, BrowserSwitchConfig config) {
+        return new BrowserSwitchAdapter(simpleClient, config);
+    }
+
+    private BrowserSwitchConfig config;
+    private BrowserSwitchClient simpleClient;
```

Comment:
Why the name `simpleClient`? Just curious if this has a greater meaning?

Reply:
Yeah this is legacy. Throwback to a time where there wasn't a better name. About to fix.